gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/domain/SweData.java

```diff
@@ -74,4 +76,12 @@ public String getJsonCracPathEsPt() {
     public String getJsonCracPathFrEs() {
         return jsonCracPathFrEs;
     }
+
+    public Crac getCrac(DichotomyDirection direction) {
```

Comment:
I dont like having logic to domain class, they should just be storage class. The logic should be done when used in my opinion. To discuss